sdks/python/scripts/process_sdk_worker.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+###########################################################################
+# This script will be run the python sdk worker.
+#
+# Positional Parameters:
+# --id=<WorkerId>
+# --logging_endpoint=<host:port>
+# --artifact_endpoint=<host:port>
+# --provision_endpoint=<host:port>
+# --control_endpoint=<host:port
+#
+# It assumes environment has pre-installed Beam and python worker code.
+# Note: No artifact staging is done. User is responsible of setting up
+# environment correctly including the user code.
+
+# Example parameters while execution --id=1 --logging_endpoint=localhost:46213 --artifact_endpoint=localhost:34075 --provision_endpoint=localhost:36555 --control_endpoint=localhost:33339
+
+export WORKER_ID=`echo $1 | cut -d'=' -f 2`
```

Comment:
The PIPELINE_OPTIONS variable is missing, see:
https://github.com/lyft/beam/commit/ae99b34ed46c5594aaf2a65c6c910f10c3d1a7cc#diff-6e5c4ac8dad73b32eb116d34d1b13deeR100

Reply:
Thanks for the link. Will update the PR